### PR TITLE
Add support of .Net Standard 2.0

### PR DESCRIPTION
**Pull request by VadimGatsura**

### CODE REVIEW

File: Tests/Linq/Linq.csproj [hunk 1]
```diff
@@ -1,197 +1,203 @@
...
-    <Reference Include="IBM.Data.Informix">
-      <HintPath>..\..\..\linq2db.2.git\Redist\IBM\IBM.Data.Informix.dll</HintPath>
-    </Reference>
-    <Reference Include="Sap.Data.Hana.v4.5">
-      <HintPath>..\..\..\linq2db.2.git\Redist\SAPHana\Sap.Data.Hana.v4.5.dll</HintPath>
-    </Reference>
-    <Reference Include="Sybase.AdoNet45.AseClient">
-      <HintPath>..\..\..\linq2db.2.git\Redist\Sybase\Sybase.AdoNet45.AseClient.dll</HintPath>
-    </Reference>
-  </ItemGroup>
```
Comment (ili) [inline Tests/Linq/Linq.csproj]: this are needed references

File: Tests/Linq/Linq.csproj [hunk 2]
```diff
@@ -1,197 +1,203 @@
...
-	<ItemGroup>
-    <None Include="..\..\Data\*">
-      <Link>Database/%(FileName)%(Extension)</Link>
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </None>
-    <None Include="..\..\Data\Create Scripts\*">
-      <Link>Database/Create Scripts/%(FileName)%(Extension)</Link>
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </None>
-  </ItemGroup>
```
Comment (ili) [inline Tests/Linq/Linq.csproj]: This were for copying scripts to output. There are `<None Update>` below, but in my case `Update` has no effect without include.

File: Source/Extensions/ReflectionExtensions.cs [hunk 3]
```diff
@@ -404,7 +404,7 @@ public static FieldInfo[] GetFieldsEx(this Type type)
 
 		public static Type[] GetGenericArgumentsEx(this Type type)
 		{
-#if NETFX_CORE && !NETSTANDARD
+#if NETFX_CORE && !NETSTANDARD && !!NETSTANDARD2_0
```
Comment (ili) [inline Source/Extensions/ReflectionExtensions.cs]: `!!NETSTANDARD2_0 `?

**Comment by VadimGatsura** [comment]
Close due to uselessness of changes. Same changes have already committed to 'version2' branch.

**Pull request closed without merging**